src/canvas.js

```diff
@@ -111,13 +111,14 @@ class CanvasCard {
     fontSize,
     font,
     initialOffset,
-    lineHeight
+    lineHeight,
+    scale
   }) {
     canvasContext.font = `${fontSize}px ${font}`;
 
     lines.forEach((line, i) => {
       const yOffset = initialOffset + lineHeight * (i + 1);
-      canvasContext.fillText(line, Config.padding, yOffset);
+      canvasContext.fillText(line, Config.padding * scale, yOffset);
```

Comment:
Could we split this into its own PR? That is, I'm not sure how this relates to changing the way we're downloading?

Reply:
done